libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/models/WalkingOptionsAdapterFactory.java

```diff
@@ -0,0 +1,25 @@
+package com.mapbox.services.android.navigation.v5.models;
+
+import com.google.gson.TypeAdapterFactory;
+import com.ryanharter.auto.value.gson.GsonTypeAdapterFactory;
+
+/**
+ * Required so that AutoValue can generate specific type adapters when needed inside the direction
+ * packages.
+ *
+ * @since 4.8.0
```

Comment:
would have loved if we could get rid of all these "since" annotations in all the model classes. The maplibre-navigation-android is quite far off the original mapbox numbers and the comments now all contain wrong information. I mean... that's probably the least of the problems in the javadocs, but hey 😀

Reply:
We can remove AutoValue in a future PR, IMHO we should start converting classes to Kotlin and the Data Model might be a good start for that. This PR only copied the models as they are.